logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/receiver/ConversationEventReceiver.kt

```diff
@@ -384,6 +384,26 @@ internal class ConversationEventReceiverImpl(
         }
     }
 
+    private suspend fun handleRenamedConversation(event: Event.Conversation.RenamedConversation) {
+        conversationRepository.updateConversationName(event.conversationId, event.conversationName, event.timestampIso)
+            .onSuccess {
+                kaliumLogger.withFeatureId(EVENT_RECEIVER).d("$TAG - The Conversation was renamed: ${event.conversationName}")
```

Comment:
Log policy is here licence and registration please 🚔
maybe it's not a great idea to log the conversation name

Reply:
@yamilmedina please censor the name of the conversation. You can ask @MohammedMokresh how to

Reply:
done !